tests/test_tables/test_records.py

```diff
@@ -19,7 +19,7 @@ def test_table_records_factory(table: Table) -> None:
 
 
 def test_table_records_lists_factory(table: Table) -> None:
-    table_records_lists = records.lists_factory(table)
+    table_records_lists = records.lists_factory(table, min_size=2)
```

Comment:
It turns out that we need the `min_size=2` here, as part of the test case itself, in order to meaningfully regression test the bug fixed here: if I reintroduce the bug and omit this constraint then it'll keep generating tables of size 1 (since that's all it could generate).

Reply:
that's strange, locally I have `table_records_lists` of various lengths without specifying `min_size`

Reply:
Did you try re-introducing the original bug? If I revert the actual behavior changes (`git checkout -f origin/master -- hypothesis_sqlalchemy/records.py`), keep the test changes, and set `min_size=1` then the updated tests still fail despite the bug having been reintroduced.

Reply:
E.g if I add statement
```python
    print(len(table_records),
          len(table.c), any(map(is_column_unique, table.c)))
```
and run tests I get something like
```
# generated records count | columns count | table has unique column
1 3 False
3 7 True
5 4 False
4 7 True
7 0 False
7 5 True
6 15 True
9 4 False
8 7 True
5 2 False
5 0 False
10 6 False
22 18 True
1 11 True
6 18 True
4 7 False
5 5 True
2 0 False
21 3 False
7 13 True
8 5 True
2 9 True
```

Reply:
oh, I've finally got it

Reply:
I haven't mentioned it anywhere, but there is a codestyle that I follow and following it keyword arguments should be passed on a separate line each like
```python
    table_records_lists = records.lists_factory(table, 
                                                min_size=2)
```
in our particular case (sorry for being scrupulous)

IMO this makes code more readable (*sparse is better than dense* afterall)